docs/coordinates/galactocentric.rst

```diff
@@ -171,7 +171,9 @@ up-to-date list of valid names, see the docstring of
 like ``'pre-v4.0'``, which sets the default parameter values to their original
 definition (i.e. pre-astropy-v4.0) values, and ``'v4.0'``, which sets the
 default parameter values to a more modern set of measurements as updated in
-Astropy version 4.0.
+Astropy version 4.0. Also, custom sets of measurements can be registered to
+`~astropy.coordinates.galactocentric_frame_defaults` and used like the
+built-in options.
 
```

Comment:
I think it would be useful to add an example to the end of the examples below demonstrating how to create a custom parameter set. I'm assuming these changes came because you are using a parameter set that is not one of the default options, so maybe you could use that set of values as your example case? :)

Reply:
You assume correctly, particularly the need to keep track of some 1-sigma errors. I'll write something up.

Reply:
done.

Reply:
It looks like you added an example to the docstring, but it would also be useful to have something in the narrative docs page. Would you be willing to add a few sentences here, or at minimum link to the docstring example?

Reply:
I've added examples and description